cmd/grafana-backups/main.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/letsencrypt/sre-tools/cmd"
+)
+
+// Query the Grafana instance by a dashboard's UID for the raw JSON and write
+// the result to a specified directory
+func writeDashboardFile(outputDirectory, uid string) error {
+	body, err := fetch("/api/dashboards/uid/" + uid)
+	if err != nil {
+		return err
+	}
+
+	return ioutil.WriteFile(outputDirectory+"/"+uid+".json", body, 0644)
```

Comment:
Small nit, I think it would be cleaner to use https://golang.org/pkg/path/#Join

_Note: If you accept my suggested edit you'll have to add `import path` as well_

```suggestion
	return ioutil.WriteFile(path.Join(outputDirectory, uid+".json"), body, 0644)
```